app/UiTPAS/UiTPASCommandValidatorServiceProvider.php

```diff
@@ -19,7 +19,8 @@ public function register(Application $app)
             function (Application $app) {
                 return new EventHasTicketSalesCommandValidator(
                     $app['uitpas'],
-                    LoggerFactory::create($app, LoggerName::forService('uitpas', 'ticket-sales'))
+                    LoggerFactory::create($app, LoggerName::forService('uitpas', 'ticket-sales')),
+                    $app['place_repository']
```

Comment:
I would always make the order of arguments depending on the "importance" of a parameter and not add newer parameters just to the end.